events/events-team/content/content-coordinator.md

```diff
@@ -0,0 +1,43 @@
+# Content Coordinator Handbook
+
+## Overview
+
+The Content Coordinator leads and manages the content programs associated with the event.  They work with the Event Lead to determine which content programs the event will have, recruits the content leads and shadows for each content program, and coordinates between the programs to make sure they all work together.  They are also a backup for any content lead who is unable to fulfill their duties, and execute any content tasks that don't fall to the content program leads.
+
+## Skills and Qualifications
+
+TODO  
+
+## Activities  
+
+The following list of activities is for a large Contributor Summit with multiple Content Programs.  For smaller summits, many of these activities will be unnecessary, and timelines can be shortened.
+
+### Weeks 1-3
+
+* Work with Event Lead to determine Content Strategy for the event.
+* Determine what Content Programs the event will have.
+* Recruit Leads & Shadows for each Content Program.
+* Post "Save the Date" placeholders for the Summit, and messages to contributors.
```

Comment:
this should be the event lead or the marketing lead to prevent folks not knowing who the point of contact is for the event/confusion

Reply:
or come from a general mailbox like contributors@

Reply:
Ah, for some reason I did it this time, so I put it in the role.  Makes sense, will remove.

Reply:
Done.